lib/std/Uri.zig

```diff
@@ -0,0 +1,515 @@
+//! Implements URI parsing roughly adhering to <https://tools.ietf.org/html/rfc3986>.
+//! Does not do perfect grammar and character class checking, but should be robust against URIs in the wild.
+
+const Url = @This();
```

Comment:
```suggestion
const Uri = @This();
```
After that there should be zero "URL" occurrences.

Reply:
Oh right, good point! :see_no_evil: